platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/DefaultProjectManager.java

```diff
@@ -1019,8 +1050,8 @@ private void deleteProjectFromWorkspace(String wsId, String projectName) throws
 
         try {
             HttpJsonHelper.request(null, link);
-        } catch (IOException | ApiException e) {
-            throw new ServerException(e);
+        } catch (IOException | ApiException exception) {
+            throw new ServerException(exception);
```

Comment:
In this case it is better to throw `new ServerException(expection.getLocalizedMessage(), exception)`